src/languages/es.ts

```diff
@@ -3610,4 +3610,21 @@ export default {
     systemMessage: {
         mergedWithCashTransaction: 'encontró un recibo para esta transacción.',
     },
+    subscriptionSize: {
```

Comment:
Please confirm translations in the `open-source` channel.

Reply:
almost all of them are already confirmed